### PR TITLE
chore: add support to python 3.11

### PR DESCRIPTION
**Pull request by csalom**
Add support to python 3.11

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,8 +1,13 @@
 Changelog
 =========
 
-1.2.0 (2022-12-16)
+Unreleased
 ----------
+* Add Python 3.111 support.
```
Comment (Danitocode) [inline CHANGELOG:6]:
```suggestion
* Add Python 3.111 support.
```

**Review by Danitocode — APPROVED** [review]
LGTM, just a minor sugestion

File: CHANGELOG (hunk 1 above)
Comment (csalom) [reply CHANGELOG]: Done!

**Review by jlmirocoll — APPROVED** [review]
LGTM :ok_hand:

**Pull request merged**